pollination/honeybee_vtk/export_images.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+from pydantic.typing import IntStr
+
+
+@dataclass
+class ExportImages(Function):
+    """Export images from am hbjson file."""
+
+    hbjson_file = Inputs.file(
+        description='HBJSON file.',
+        path='input.hbjson'
+    )
+
+    name = Inputs.str(
+        description='Name of image files.',
+        default='Camera'
+    )
+
+    folder = Inputs.folder(
+        description='Target folder where the images will be written.',
+        path='output_folder',
+        default='.'
+    )
+
+    image_type = Inputs.str(
+        description='Choose the type of image file.',
+        default='jpg'
+    )
+
+    image_width = Inputs.int(
+        description='Width of images in pixels.',
+        default=2500
+    )
+
+    image_height = Inputs.int(
+        description='Height of images in pixels.',
+        default=2500
+    )
```

Comment:
Shouldn't we get these two properties from the view information itself? Or did we take `-x` and `-y` values out of the view object. This is fine for now but it will become a major limitation down the road if all the images should be the same size.

Reply:
Nice! Are you referring to the `dimension_x_y` method? I think you are. I did not see this method until you mentioned this. I can start using it for hbjsons with radiance views. For hbjsons without radiance views, we need to agree on some numbers for the width and height of images. The default cameras will use those numbers.

Reply:
Got it! Then in that case we should rename this to heights and width for default views or something like that.

I don't remember what was `dimension_x_y` but that sounds like what you need.

Reply:
I captured it in an [issue](https://github.com/ladybug-tools/honeybee-vtk/issues/191) so I can get it done later.